setup.py

```
@@ -1,17 +1,19 @@
 
-with open("README.md", "r", encoding="utf-8") as fh:
+from pathlib import Path
+readme_path = Path(__file__).parent / "README.md"
+with open(readme_path, "r", encoding="utf-8") as fh:
     long_description = fh.read()
 
 setup(
     name='robotick',
     version='0.1.0',
-    description='A modular Python framework for robotics and control systems, supporting real-time workloads and simulation.',
+    description='A modular C++ / Python framework for robotics and control systems, supporting real-time workloads and simulation.',
```

Pylint (3.3.7)

[convention] Line too long (132/100)

(C0301)